```sql
SELECT
t1.`ATTENDANCE_RATIO`,
t1.`CREATE_DATE`,
t1.`CREATE_UID`,
t1.`DEFAULT_FLAG`,
t1.`END_START`,
t1.`ID`,
t1.`INVALID_OVERTIME`,
t1.`NAME`,
t1.`ORDER_VALUE`,
t1.`OVERTIME_VALID`,
t1.`PUBLIC_FLAG`,
t1.`REST_ENABLE`,
t1.`REST_START`,
t1.`RULE_ID`,
t1.`RULE_NAME`,
t1.`START_OVERTIME`,
t1.`WORKING_NUMBER`,
t1.`WRITE_DATE`,
t1.`WRITE_UID`
FROM `ATTENDANCE_SHIFT` t1 


```